woocommerce/pinterest-for-woocommerce
# Add method to check if billing is setup for user.

### Changes proposed in this Pull Request:

Adds method that uses the `billing_profiles` endpoint to get the user billing status information.



Closes #541.


### Detailed test instructions:


1. Connect to Pinterest
2. Use the method somewhere in the code and observe the return value


### Additional details:
This should replace the use of billing_data endpoint.


### Changelog entry

> Add - Method for obtaining billing status information.
